google/cloud/bigtable/internal/bulk_mutator.cc

```diff
@@ -197,14 +197,42 @@ grpc::Status BulkMutator::MakeOneRequest(bigtable::DataClient& client,
   // Read the stream of responses.
   btproto::MutateRowsResponse response;
   while (stream->Read(&response)) {
-    state_.OnRead(response);
+    state_.OnRead(std::move(response));
   }
   // Handle any errors in the stream.
   auto grpc_status = stream->Finish();
   state_.OnFinish(MakeStatusFromRpcError(grpc_status));
   return grpc_status;
 }
 
+Status BulkMutator::MakeOneRequest(bigtable_internal::BigtableStub& stub) {
+  // Send the request to the server.
+  auto const& mutations = state_.BeforeStart();
+
+  // Configure the context
+  auto const& options = google::cloud::internal::CurrentOptions();
+  auto context = absl::make_unique<grpc::ClientContext>();
+  google::cloud::internal::ConfigureContext(*context, options);
+
+  struct UnpackVariant {
+    BulkMutatorState& state;
+    bool operator()(btproto::MutateRowsResponse&& r) {
+      state.OnRead(std::move(r));
+      return true;
+    }
+    bool operator()(Status&& s) {
+      state.OnFinish(std::move(s));
+      return false;
+    }
```

Comment:
If these work with `&&` then they should work just as well without them too.

Reply:
Fixed.

FWIW, I copied this: https://github.com/googleapis/google-cloud-cpp/blob/41d558bcd48aaba612f49d955d83db83d6f226e1/google/cloud/stream_range.h#L182-L194